pygeoapi/api.py

```diff
@@ -3393,12 +3394,6 @@ def execute_process(self, request: Union[APIRequest, Any],
                 HTTPStatus.NOT_FOUND, headers,
                 request.format, 'NoSuchProcess', msg)
 
-        if not self.manager:
-            msg = 'Process manager is undefined'
-            return self.get_exception(
-                HTTPStatus.INTERNAL_SERVER_ERROR, headers,
-                request.format, 'NoApplicableCode', msg)
-
```

Comment:
Removed this bit because it does not apply - as per this class' `__init__()` method, there is always a manager, so it is no longer relevant to test if `self.manager` exists